pkg/custmsg/custom_message.go

```diff
@@ -117,7 +117,7 @@ func sendLogAsCustomMessageW(msg string, labels map[string]string) error {
 
 	err = beholder.GetEmitter().Emit(context.Background(), payloadBytes,
 		"beholder_data_schema", "/beholder-base-message/versions/1", // required
-		"beholder_domain", "chainlink", // required
+		"beholder_domain", "platform", // required
 		"beholder_entity", "BaseMessage", // required
```

Comment:
Why do we use underscores instead of dot separation for these?

Reply:
@cll-gg may know, the naming of these keys was decided by the Beholder team